### ISSUE TITLE
Provide a F-Droid build

### ISSUE THREAD

**Issue by alexanderadam** [rahulkumar2048/Performance-Tweaker#7]
Is the Performance Tweaker ready to use?

If yes could you add it to [F-Droid](https://f-droid.org/repository/browse/)?

Thank you very much

**Comment by rahulkumar2048** [comment]
will soon upload it.

**Comment by alexanderadam** [comment]
Cool thing, thank you!

**Comment by IzzySoft** [comment]
Two years later: How soon is "soon"? Maybe at least tag a release and attach an `.apk`? :wink:

**Comment by rahulkumar66** [comment]
@IzzySoft Done added the release with the apk

**Comment by IzzySoft** [comment]
@rahulkumar66 Thanks a lot! If you don't mind, I could now add it to [my F-Droid compatible repo](https://apt.izzysoft.de/fdroid), which then would fetch new releases automatically within ~24h of your tagging them (provided an `.apk` file is attached to the corresponding tag – while the tag name should correspond to the `versionName` declared in the manifest, e.g. the current tag `v0.1.11` would mean a `versionName = '0.1.11'`, and the `versionCode` must be increased with each new tagged version). That way folks without Playstore have a convenient way to install your app and keep it updated without any fuss. Also helps to "spread the word" :wink: Would you agree?

**Comment by rahulkumar66** [comment]
sure thing.
let me know after you have added my app to your repo

**Comment by IzzySoft** [comment]
[Here you go](https://apt.izzysoft.de/fdroid/index/apk/com.performancetweaker.app). If I put anything wrong in the description let me know to correct it. The Wakelock part (in the Playstore screenshots) somehow reminded me of BBS – by the libs used I see I wasn't that far off :wink: Nice combination of tools to have in a tweaking app, congrats!

**Comment by rahulkumar66** [comment]
Thanks.
Will be adding even more features.

**Comment by Kelvino9** [comment]
@rahulkumar66 can you please tag the latest versions for release? We are still on 0.1.11 on F-Droid. Really love the app. It works great in its initial version. Can't wait to try the latest version.

**Comment by IzzySoft** [comment]
@rahulkumar66 Indeed! As I wrote above, my auto-updater picks new versions within 24h ***after*** your attaching them to a new tag. It cannot do so before :wink:

**Comment by IzzySoft** [comment]
@rahulkumar66 according to `build.gradle`, this app is already at v0.2.9 – but the last tag is v0.2.5. Would you consider adding a tag at least for the latest version, and attach the APK to it – so it can be updated at least in my repo? Thanks!
